modules/user/server-ts/social/facebook/index.js

```diff
@@ -70,14 +70,16 @@ if (settings.user.auth.facebook.enabled && !__TEST__) {
       const user = await User.getUser(req.user.id);
       const redirectUrl = req.query.state;
       const tokens = await access.grantAccess(user, req);
+      const currentUser = await getCurrentUser(req, res);
```

Comment:
Why did you add this line? There was not any `currentUser` data passed previously.

Reply:
done